duet/sys/config.g

```diff
@@ -72,26 +70,19 @@ M106 P1 S0
 M106 P2 S0
 
 ; Tool definitions
-M563 P0 D0 H1                           ; Define tool 0
-G10 P0 S0 R0                            ; Set tool 0 operating and standby temperatures
-;*** If you have a single-nozzle build, leave the next 2 lines commented out.
-;M563 P1 D1 H2                          ; Define tool 1
-;G10 P1 S0 R0 X0 Y17                    ; Set tool 1 operating and standby temperatures
-
-; Z probe and compensation definition
+M563 P0 S"config-user not set" D0 H1    ; Define tool 0
```

Comment:
Its pretty safe regardless of build to set the tool, isn't it? It always seemed pretty generic to me.

Reply:
Ah right. That slipped through :) 
I'd done that before you'd done your bits (which were cool btw :+1:  )

Could make it "M563 tool config not set" :) or just remove, I'm easy.

Reply:
defined as "Tool 0" otherwise left in as default with -273C as default temps.
I hope thats 0K :rofl: